src/testing/tree_construction/parser.rs

```diff
@@ -306,7 +308,8 @@ fn test(i: Span) -> IResult<Span, TestSpec> {
 }
 
 pub fn parse_str(i: &str) -> Result<Vec<TestSpec>> {
-    let input = i.to_owned() + "\n";
+    // Deal with a corner case that makes it hard to parse tricky01.dat.
+    let input = i.replace("\"\n\n\"", QUOTED_DOUBLE_NEWLINE).to_owned() + "\n";
```

Comment:
The challenge with this one is that every test but this one is delimited by `\n\n`, which is what is currently being used as the delimiter between tests.